src/compiler/build/watch-build.ts

```diff
@@ -104,7 +111,7 @@ export const createWatchBuild = async (
           break;
       }
 
-      config.logger.debug(`onFsChange ${eventKind}: ${p}`);
+      console.log(`🔥:watchBuild:onFsChange type=${eventKind}, path=${p}, time=${new Date().getTime()}`);
       tsWatchProgram.rebuild();
     }
```

Comment:
Can we also log when this conditional above evaluates to `false`? e.g. https://github.com/ionic-team/stencil/compare/rwaskiewicz/ts-reload-bug-debug#diff-b3739ad1981b6aaadc57ffe972513cf24b1cda208aceccd8a642b0539fc8e878R112